### PR TITLE
fix: Add container name to metrics to exclude metrics which are not related

### PR DESCRIPTION
**Pull request by jeremyharisch**
**Description**

Changes proposed in this pull request:

- Added container name to max parallel workers metrics dashboard to represent the actual wanted workers we want to have at the dashboard. Because before the metrics from `runtime-monitoring-operator` have been included as well
- 

**Before:**
Screenshot 2024-01-29 at 15 00 56

**After:**

Screenshot 2024-01-29 at 15 00 40

Actuel configured workers
Screenshot 2024-01-29 at 15 03 59

### CODE REVIEW

**Comment by jeremyharisch** [comment]
Screenshot 2024-01-29 at 15 04 57

If we remove the sum function then we can actually see why kyma was 85 before, but since we are just interested in the KLM metrics, we should add the container name

**Review by nesmabadr — APPROVED** [review]

File: config/grafana/overview.json [hunk 1]
```diff
@@ -1994,5 +1995,5 @@
   "timezone": "",
   "title": "Lifecycle Manager Overview",
   "uid": "O3DH7uunk",
-  "version": 7
-}
+  "version": 1
```
Comment (nesmabadr) [inline config/grafana/overview.json:1998]: is this version correct? It was 7

Comment (jeremyharisch) [reply config/grafana/overview.json:1998]: Was set by Plutono itself. I just changed the two lines with adding the container name. The rest was done by plutono

Comment (jeremyharisch) [reply config/grafana/overview.json:1998]: So I guess, it should be correct

Comment (nesmabadr) [reply config/grafana/overview.json:1998]: Yeah I was just surprised that it was downgraded so was just making sure you didn't edit it by mistake

**Pull request merged**